apps/coscrad-frontend/src/components/resources/playlists/playlist-index.presenter.tsx

```diff
@@ -0,0 +1,30 @@
+import { IPlayListViewModel } from '@coscrad/api-interfaces';
+import { PlaylistIndexState } from '../../../store/slices/resources/playlists/types';
+import { HeadingLabel, IndexTable } from '../../../utils/generic-components/presenters/tables';
+import { renderAggregateIdCell } from '../utils/render-aggregate-id-cell';
+
+import { CellRenderersDefinition } from '../../../utils/generic-components/presenters/tables/generic-index-table-presenter/types/cell-renderers-definition';
+import { renderMultilingualTextCell } from '../utils/render-multilingual-text-cell';
+
+export const PlaylistIndexPresenter = ({ entities: playlists }: PlaylistIndexState) => {
+    const headingLabels: HeadingLabel<IPlayListViewModel>[] = [
+        { propertyKey: 'id', headingLabel: 'link' },
+        { propertyKey: 'episodes', headingLabel: 'episode' },
```

Comment:
Make the heading 'Number of Episodes' or just 'Episodes'. Then render the number of episodes in the data cells.